### ISSUE TITLE
[review] Figure 12

### ISSUE THREAD

**Issue by ppxasjsm** [alchemistry/alchemical-best-practices#97]
Section 8.5 and Figure 12: Here I would again mention the use of both forward and reverse cumulative averages and plot them explicitly in Figure 12.

Is this possible to do? Is the data available?
[Figure.pdf](https://github.com/alchemistry/alchemical-best-practices/files/5291790/Figure.pdf)

**Comment by andrrizzi** [comment]
I found some data that might be usable. What do you think about this? https://github.com/alchemistry/alchemical-best-practices/blob/update_fig_12/paper/figures/fig9_convergence/Figure.pdf

I have included in the plot a forward/reverse free energy trajectory for a single replicate. We could use it to show that the test can detect lack of convergence.

**Comment by davidlmobley** [comment]
Looks good, yes!

**Comment by ppxasjsm** [comment]
Great! Could the reference dashed line maybe not also be dashed? Maybe a straight one or dashed dotted or something to show that this is the reference?

**Comment by andrrizzi** [comment]
> Maybe a straight one or dashed dotted or something to show that this is the reference?

Done! I went with the dash-dot line to make it different from the free energy trajectories. I have also updated the caption, the text in the manuscript, and the review document.

I have manually uploaded on overleaf the Figure.pdf file, but I haven't manually updated the code/data on overleaf since I suspect that will happen when it will be synched with github after the PR has been merged.
